packages/material-ui/src/Box/Box.js

```diff
@@ -9,7 +9,7 @@ import sizing from '@material-ui/system/sizing';
 import spacing from '@material-ui/system/spacing';
 import typography from '@material-ui/system/typography';
 import css from '@material-ui/system/css';
-import { styled } from '@material-ui/styles';
+import styled from '../styles/styled';
```

Comment:
```suggestion
import styled from '../styled';
```

Reply:
@oliviertassinari 
```bash
./material-ui$ cd packages/material-ui/src/Box/
./material-uipackages/material-ui/src$ cd ../
./material-uipackages/material-ui/src$ cd styled
bash: cd: styled: No such file or directory
```

Reply:
Ok, of course, we are in the core, not stlyes.

Reply:
What about all our imports (>100)
```jsx
import withStyles from '../styles/withStyles';
```
Should we change them too?

Reply:
Ok I'm probably wrong. I was thinking about #14888 but that doesn't apply here. Relative imports from >2nd level should be fine because at this point we're already in `build/esm`. It's probably better for build times since `webpack` doesn't have to check the `package.json` again.


PR is fine as is :+1:

Reply:
@oliviertassinari Thanks for the follow-up!

Reply:
Thanks for the feedback!

One question @eps1lon , just for my interest, where can I read more about the ```build/esm``` stuff?

Reply:
This should give you a good overview
- [original issue](https://github.com/mui-org/material-ui/issues/11281)
- [complete es-module build implementation](https://github.com/mui-org/material-ui/pull/13391)
- [follow-up regarding 2nd level imports](https://github.com/mui-org/material-ui/pull/14888)

For more questions please open a topic on our [spectrum](https://spectrum.chat/material-ui).